js/screens/CompetitionList/CompetitionList.js

```diff
@@ -33,8 +43,9 @@ export default CompetitionList;
 
 CompetitionList.propTypes = {
   list: PropTypes.array.isRequired,
+  navigation: PropTypes.object.isRequired
 };
 
 CompetitionList.defaultProps = {
-  list: [],
-}
+  list: []
```

Comment:
if your propType is required, you don't need default prop for that. If you have questions, please ask.